src/components/TableCard/TableCard.jsx

```diff
@@ -90,8 +91,8 @@ const StyledStatefulTable = styled(({ showHeader, isExpanded, data, ...rest }) =
       height: 3rem;
 
       th {
-        padding-top: 5px;
-        padding-bottom: 10px;
+        padding-top: ${spacing02};
+        padding-bottom: ${spacing03 - 2};
```

Comment:
Same comment as above. There is not need to use a token if you have to add or subtract a value from the token. We should try to standardize or we can leave as is. If there is any question to a UX visual question we can ask it on the #iot-pal slack channel.